### ISSUE TITLE
The "unique" column option is not supported. 

### ISSUE THREAD

**Issue by driveitlikearocket** [nextcloud/maps#629]
upon trying to install from appstore on relatively fresh install of Nextcloud 21.0.2, I get the error mentioned in the title:
The "unique" column option is not supported. 

LMK if there's anything I can provide to help with this issue

**Comment by dem-ars** [comment]
Uninstall your app from market and install maps 1.9 nightly.

**Comment by albjeremias** [comment]
how do u install a nightly? :|

**Comment by DecaTec** [comment]
Disable and remove the maps app from within Nextcloud.
Next, on the command line:
```
cd
wget https://github.com/nextcloud/maps/releases/download/v0.1.9-8-nightly/maps-0.1.9-8-nightly.tar.gz
tar -xf maps-0.1.9-8-nightly.tar.gz
cp -r maps /var/www/nextcloud/apps/maps/
chown -R www-data:www-data /var/www/nextcloud/apps/maps
```

Then go into the apps category in Nextcloud again and enable the maps app.

**Comment by jedie** [comment]
When will the marked version fixed?

**Comment by tacruc** [comment]
Fixed by #568
